code/style.css

```diff
@@ -2,6 +2,307 @@
+}
+
+section {
+  padding: 6rem, 2rem;
+	background: #D3DEDC;
+}
+
+/* =================
+sign up
+==================== */
+@import url('https://fonts.googleapis.com/css2?family=Montserrat:wght@400;700&display=swap');
+
+.registration_form{
+	background: #4B6587;
+	padding: 30px;
+	border-radius: 5px;
+	width: 400px;
+  height: 457px;
+  align-items: center;
+  margin-right: 20px;
+  margin-top: 40px;
+  margin-bottom: 40px;
+}
+
+.registration_form .title{
+	text-align: center;
+	font-size: 20px;
+	text-transform: uppercase;
+	color: #30475E;
+	letter-spacing: 5px;
+	font-weight: 700;
+}
+
+.form_wrap{
+	margin-top: 35px;
+}
+
+.form_wrap .input_wrap{
+	margin-bottom: 15px;
+}
+
+.form_wrap .input_wrap:last-child{
+	margin-bottom: 0;
+}
+
+.form_wrap .input_wrap label{
+	display: block;
+	margin-bottom: 3px;
+	color: #9597a6;
+}
+
+.form_wrap .input_grp{
+	display: flex;
+	justify-content: space-between;
+}
+
+.form_wrap .input_grp  input[type="text"]{
+	width: 165px;
+}
+
+.form_wrap  input[type="text"]{
+	width: 100%;
+	border-radius: 3px;
+	border: 1px solid #9597a6;
+	padding: 10px;
+	outline: none;
+}
+
+.form_wrap  input[type="text"]:focus{
+	border-color: #ebd0ce;
+}
+
+.form_wrap ul{
+	background: #fff;
+	padding: 8px 10px;
+	border-radius: 3px;
+	display: flex;
+	justify-content: center;
+}
+
+.form_wrap ul li:first-child{
+	margin-right: 15px;
+}
+
+.form_wrap ul .radio_wrap{
+	position: relative;
+	margin-bottom: 0;
+}
+
+.form_wrap ul .radio_wrap .input_radio{
+	position: absolute;
+	top: 0;
+	right: 0;
+	opacity: 0;
+}
+
+.form_wrap ul .radio_wrap span{
+	display: inline-block;
+	font-size: 14px;
+	padding: 3px 20px;
+	border-radius: 3px;
+	color: black;
+}
+
+.form_wrap .input_radio:checked ~ span{
+	background: #7E8A97;
+}
+
+.form_wrap .submit_btn{
+	width: 100%;
+	background: #8E806A;
+	padding: 10px;
+	border: 0;
+	border-radius: 3px;
+	text-transform: uppercase;
+	letter-spacing: 3px;
+	cursor: pointer;
 }
+
+.form_wrap .submit_btn:hover{
+	background: #FFE6BC;
+}
+
+.input_wrap {
+  color:#9597a6;
+  margin-bottom: 15px;
+}
+
+/* ==================
+accordion
+===================== */
+
+  .accordion .title{
+   text-align: center;
+   font-size: 20px;
+   text-transform: uppercase;
+   color: #30475E;
+   letter-spacing: 5px;
+   font-weight: 700;
+  }
+
+  .accordion {
+    background: #4B6587;
+    padding: 25px;
+    border-radius: 5px;
+    width: 400px;
+    height: 457px;
+    margin-left: 20px;
+    margin-top: 40px;
+    margin-bottom: 40px;
+  }
+  
+  .label {
+    border-radius: 3px;
+  }
+  .accordion .contentbox_1 {
+    position: relative;
+    margin: 10px 20px;
+  }
+  .accordion .contentbox_1 .label {
+    position: relative;
+    padding: 10px;
+    background: #2694af;
+    color: #fff;
+    cursor: pointer;
+  }
+
+  .accordion .contentbox_1 .label::before {
+    content: '+';
+    position: absolute;
+    top: 50%;
+    right: 20px;
+    transform: translateY(-50%);
+    font-size: 1.5em;
+  }
```

Comment:
Great job! This function was complicated to achieve (and I think it is black level as well) so well done!!